hashes/src/sha256.rs

```diff
@@ -882,7 +882,6 @@ mod tests {
     }
 
     #[test]
-    #[cfg(feature = "alloc")]
```

Comment:
Oh, you didn't bother to fix that commit, nevermind. But would be nice to squash those two in case you make any other changes.

Reply:
Leaving as is, we have two acks. Thanks.